app/views/layouts/_header.html.erb

```diff
@@ -68,23 +68,23 @@
         <li><%= link_to 'Employer Services', employers_path %></li>
         <li><%= link_to 'Web Development', deploy_path %></li>
         <li role="separator" class="divider"></li>
-      
+
         <li><%= link_to 'Success Stories', success_stories_path %></li>
         <li><%= link_to 'Blog', 'https://medium.com/operation-code' %></li>
         <li role="separator" class="divider"></li>
-        
+
         <a class="btn" href="https://donorbox.org/operationcode"
             data-constrainwidth="false" data-beloworigin="true" data-activates="Donate">Donate</a>
-            
+
         <li role="separator" class="divider"></li>
 
         <% if veteran_signed_in? %>
           <li><%= link_to 'Profile', profile_path %></li>
           <li><%= link_to 'Sign Out', destroy_veteran_session_path, method: :delete %></li>
         <% else %>
           <li>
-            <% ab_test(:signup_button_size, 'btn btn-large', '') do |button_size| %>
-            <%= link_to 'Join', new_veteran_registration_path, class: button_size, style: 'background-color: #B23900;' %>
+            <% ab_test(:signup_button_size, 'btn btn-large', 'btn') do |button_size| %>
+            <%= link_to 'Join', new_veteran_registration_path, class: 'btn', style: 'background-color: #B23900;' %>
```

Comment:
This will still break the A/B test. It is testing the size of the 'Join Now' button. `button_size` will alternate between `btn btn-large` and `btn` via the variable passed to the block (`button_size`). 

What you have here will always hard code the class (the subject of the test) to `btn` rendering the test useless. 

Can you please change the class back to `button_size`?

This change is also present in #640. If you're able to can you please remove those commits from that PR? Feel free to DM me on slack for assistance.